markdown-pages/.gitignore

```diff
@@ -0,0 +1,4 @@
+# Ignore everything in this directory
```

Comment:
Explain the reason for more details.

Reply:
This is a temporary workaround to pass the build. I think the complete SIG component will be removed in the near future.